src/content/blog/astro-4140.mdx

```diff
@@ -0,0 +1,271 @@
+
+### Sharing your loaders
+
+Loaders are better when they're shared! You can create a module that exports a loader and publish it to npm, and then anyone can use it on their site. Tag your packages with the keyword [`astro-loader`](https://www.npmjs.com/search?q=keywords:astro-loader) so they can be found. We're excited to see what the community comes up with! To get started, [take a look at some examples](https://github.com/ascorbic/astro-loaders/). Here's how to load content using an RSS/Atom feed loader:
+
+```ts
+// src/content/config.ts
+import { defineCollection } from "astro:content";
+import { feedLoader } from "@ascorbic/feed-loader";
+
+const podcasts = defineCollection({
+  loader: feedLoader({
+    url: "https://feeds.99percentinvisible.org/99percentinvisible",
+  }),
+});
+
+export const collections = { podcasts };
+```
+
+### Learn more
+
+To find out more about using the Content Layer API, check out [the Content Layer RFC](https://github.com/withastro/roadmap/blob/content-layer/proposals/0047-content-layer.md) and [share your feedback](https://github.com/withastro/roadmap/pull/982).
+
+## Experimental: Intellisense inside content files
+
+Astro 4.14 introduces experimental support for Intellisense inside the frontmatter of your content. This feature helps you write content files more efficiently by providing completions, validation, hover information, and more for frontmatter keys and values. These tools are based on your content schemas, and available directly in your editor.
+
+This feature is available in VS Code and other editors that support the Language Server Protocol for Markdown, MDX, and Markdoc files.
+
+<BlogContentImage src={frontmatterIntellisense} />
+
+To enable content file intellisense, add the following to your Astro config:
+
+```js
+import { defineConfig } from 'astro';
+
+export default defineConfig({
+	experimental: {
+		contentIntellisense: true
+	}
+})
+```
+
+and in VS Code, enable the `astro.content-intellisense` setting. For other editors, pass `contentIntellisense: true` to the initialization parameters of the Astro language server.
+
+```json
+{
+	"astro.content-intellisense": true
+}
+```
+
+This feature currently only works in the editor, and will be enabled in `astro check` in a future release.
+
+## Deprecate support for dynamic `prerender` values
+
+In order to improve our bundling and tree-shaking capabilities, Astro 4.14 deprecates support for using dynamic values for the `prerender` option in your Astro pages.
+
+This feature can now be replaced with the new `astro:route:setup` hook in Astro integrations, which allows you to dynamically control the prerendering of specific pages.
+
+```diff
+---
+- export const prerender = import.meta.env.PROD;
+---
+```
```

Comment:
We have Expressive Code set up here, which means it allows us to use diff syntax + syntax highlighting.
````suggestion
```diff lang="astro"
---
- export const prerender = import.meta.env.PROD;
---
```
````